Test/DependencyCollector/FunctionalTests/TestApps/Aspx451/ExternalCalls.aspx.cs

```diff
@@ -22,7 +22,7 @@ public partial class ExternalCalls : System.Web.UI.Page
         /// <summary>
         /// Valid SQL Query.
         /// </summary> 
-        private const string ValidSqlQueryToApmDatabase = "select top 2 * from dbo.Messages";
+        private const string ValidSqlQueryToApmDatabase = "WAITFOR DELAY '00:00:00:001'; select * from dbo.Messages";
```

Comment:
There is no race condition here.
The product cannot measure dependency below 1 msec, and the actual time as <1msec. This delay will ensure a minimum 1msec dependency duration.